src/test/java/org/jenkinsci/plugins/ConfigurableExtensionFilterTest.java

```diff
@@ -2,36 +2,67 @@
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.fail;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
-import org.jenkinsci.plugins.ConfigurableExtensionFilter.DescriptorImpl;
-import org.junit.Rule;
-import org.junit.Test;
+import hudson.ExtensionList;
+import org.csanchez.jenkins.plugins.kubernetes.pod.retention.Reaper.TerminateAgentOnImagePullBackOff;
+import org.csanchez.jenkins.plugins.kubernetes.pod.retention.Reaper.TerminateAgentOnPodFailed;
+import org.junit.jupiter.api.Test;
 import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.junit.jupiter.WithJenkins;
 
+@WithJenkins
 public class ConfigurableExtensionFilterTest {
 
-    @Rule
-    public JenkinsRule j = new JenkinsRule();
-
     @Test
-    public void nullSafety() {
-        DescriptorImpl impl = new DescriptorImpl();
+    public void nullSafety(JenkinsRule j) {
+        ConfigurableExtensionFilter.DescriptorImpl impl =
```

Comment:
If we use JenkinsRule we need to check with the desciptor from JenkinsRule and not an object created directly in the test